shoes-swt/lib/shoes/swt/redrawing_aspect.rb

```diff
@@ -99,6 +100,14 @@ def redraw_area(left, top, width, height, include_children = true)
         end
       end
 
+      def redraw_target(element)
+        if element.gui
+          element.gui.redraw_target
+        else
+          element
+        end
+      end
```

Comment:
I can only assume that this was done because sometimes we get DSL elements here, sometimes we get backend elements? Might be worth unifying if that's the case.. not here though probably in another ticket.